.github/run_test.sh

```diff
@@ -8,48 +8,49 @@ C7_CONTAINER_IMAGE="quay.io/linux-system-roles/c7-network-role"
 C9S_CONTAINER_IMAGE="quay.io/linux-system-roles/c9s-network-role"
 PODMAN_OPTS="--systemd=true --privileged"
 
-# FIXME: test_wireless has been removed because is not supported on CI
-# container EL7. We need to add this back for EL8 or greater.
-EXCLUDE_TESTS='
+# exclude bond tests since missing the bonding kernel module
```

Comment:
I am not sure if this is the real reason. Isn't c9s using the same ubuntu kernel? Why is it failing on c7/c8s but not on c9s? Thanks!

Reply:
```
RUN_PLAYBOOK_WITH_INITSCRIPTS = """# SPDX-License-Identifier: BSD-3-Clause
# This file was generated by ensure_provider_tests.py
---
- hosts: all
  name: Run playbook '{test_playbook}' with initscripts as provider
  tasks:
    - include_tasks: tasks/el_repo_setup.yml
    - name: Set network provider to 'initscripts'
      set_fact:
        network_provider: initscripts
      tags:
        - always

- import_playbook: {test_playbook}
  when: (ansible_distribution in ['CentOS','RedHat'] and\n    \
ansible_distribution_major_version | int < 9)
"""
```
Initscripts provider tests are skipped for rhel-9 and centos-9.